### PR TITLE
Kyvon Thompsons Read ME

### PR DESCRIPTION
**Pull request by KRose77**

### CODE REVIEW

File: lesson_00/Kyvonreadme.md
Comment (anthonydmays) [inline lesson_00/Kyvonreadme.md:1]: You've put the file in the wrong place. Your README.md file should be located in the /lesson_00/kyvon/ directory.

File: lesson_00/Kyvonreadme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Learn about Kyvon Thompson
+
+# My Niche
+I'm a Delaware and New York based music video director and editor with skills in VFX, videography, photography, lighting and mixed media. This has led me to work and collab with and help develop the visual articulation of many talented artists and other creators either in Delaware or New york.
+
+## Examples of my work
+![EX1 Medium](https://github.com/KRose77/code-differently-24-q1/assets/160963028/17c4fb8d-bc5d-4d6f-81ea-7b39a3e51d88)
+
+
+[![](https://markdown-videos-api.jorgenkh.no/youtube/814SngdguwQ)](https://youtu.be/814SngdguwQ)
```
Comment (anthonydmays) [inline lesson_00/Kyvonreadme.md:10]: Link is broken when I view it.

File: lesson_00/Kyvonreadme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#### Favorite Food
+1. Birria Tacos
+2. Churros
+3. Pizza
+4. Waffles
+5. French Toast
+6. Chicken Rasta Pasta
+
+#### How you can reach me or see more of my work
+You can reach me primarily by email or linkedin and you can see more of my work on instagram @rosevisualsss
```
Comment (anthonydmays) [inline lesson_00/Kyvonreadme.md:28]: nit: Should be spelled "LinkedIn", not "linkedin". Fix the casing of instagram as well and other proper nouns.

File: lesson_00/Kyvonreadme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Learn about Kyvon Thompson
```
Comment (anthonydmays) [inline lesson_00/Kyvonreadme.md:1]: This is a decent start, but you should add something about yourself that will help other collaborate with you professionally (e.g. what are your best working hours? how do you prefer that teammates communicate with you?).

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Thanks for sending! Please see comments and make changes.

**Review by KRose77 — COMMENTED** [review]
Fixed casing.

File: lesson_00/Kyvon/Kyvonreadme.md
Comment (anthonydmays) [inline lesson_00/Kyvon/Kyvonreadme.md:1]: The name of the file is incorrect (should be README.md).

File: lesson_00/Kyvon/Kyvonreadme.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+#### Favorite Food
+1. Birria Tacos
+2. Churros
+3. Pizza
+4. Waffles
+5. French Toast
+6. Chicken Rasta Pasta
+
+#### How you can reach me or see more of my work
+You can reach me primarily by email or LinkedIn @KyvonThompson and Rosevisualsss52@gmail.com and you can see more of my work on Instagram @rosevisualsss
```
Comment (anthonydmays) [inline lesson_00/Kyvon/Kyvonreadme.md:31]: More than welcome to use your real email address here, but use caution. Just making sure to mention that here.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**